### PR TITLE
feat(Rows): a11y update

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1955)

- Make detail/right content readable by screen readers
- Include the entire row as part of the control's content (switch/checkbox/radio)

### CODE REVIEW

File: src/list.tsx [hunk 1]
```diff
@@ -362,9 +366,18 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
...
     // iOS voiceover reads links with multiple lines as separate links. By setting aria-label and marking content as aria-hidden, we can make it read the whole row as one link.
-    const computedAriaLabelForLink = [title, headlineText, subtitle, description, extraText, detail]
+    const computedAriaLabelForLink = [
+        title,
+        headlineText,
+        subtitle,
+        description,
+        extraText,
+        detail,
+        rightText,
```
Comment (marcoskolodny) [inline src/list.tsx:379]: Right content was missing here

File: src/list.tsx [hunk 2]
```diff
@@ -106,11 +108,13 @@ export const Content: React.FC<ContentProps> = ({
     const centerY = numTextLines === 1;
 
     return (
-        <Box paddingY={16} className={styles.content}>
+        <Box paddingY={16} className={styles.content} id={labelId}>
```
Comment (marcoskolodny) [inline src/list.tsx:111]: Moving the labelId to the entire row. In this way, detail/right is included inside control's content

**Review by atabel — APPROVED** [review]

**Review by aweell — APPROVED** [review]
Nice 🥇

**Review by pladaria — APPROVED** [review]

**Pull request merged**